hazelcast/src/main/java/com/hazelcast/sql/SqlResult.java

```diff
@@ -41,33 +41,24 @@
  * }
  * </pre>
  *
- * <p>
  * <h4>Usage for update count</h4>
  *
  * <pre>
  *     long updated = hazelcastInstance.getSql().execute("UPDATE ...").updateCount();
  * </pre>
  *
  * You don't need to call {@link #close()} in this case.
+ *
+ * <h4>Determining the type of the result</h4>
+ *
+ * To determine whether the result contains rows or an update count, call
+ * {@link #updateCount()}. If it returns -1, the result is rows, otherwise
+ * it's update count.
  */
 public interface SqlResult extends Iterable<SqlRow>, AutoCloseable {
 
     /**
-     * If this result represents a row set, this method returns {@code false}.
-     * If this result represents an update count (such as for a DML query), it
-     * returns {@code true}.
-     *
-     * @return {@code false} for a rows result and {@code true} for an update
-     *     count result
-     */
-    boolean isUpdateCount();
```

Comment:
I think that instead of asking the user to do `updateCount() == -1` to check for the result type, we should leave this method in the inversed form. This is more convenient and easier to use, and makes no harm to the API:
```
boolean isResultSet() {
    return updateCount() == -1;
}
```

Reply:
Agree, fixed.